pkgs/by-name/py/pyflyby/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

```suggestion
python3.pkgs.buildPythonApplication (finalAttrs: {
```
- https://nix.dev/guides/best-practices#recursive-attribute-set-rec
- #315337

---

i think fixed-point arguments is currently unsupported for `buildPython*`

https://github.com/NixOS/nixpkgs/pull/271387

---

In that case, at least use `let in`; `rec` is almost always bad